src/community/mod.rs

```diff
@@ -67,6 +119,7 @@ pub struct Community {
     pub wiki1: Option<WikiPage>,
     pub wiki2: Option<WikiPage>,
     pub features: CommunityFeatureFlags,
+    pub addon_list: Vec<CommunityAddOnConfigId>,
```

Comment:
Let's use `Vec<CommunityAddOnConfig>` as in order to render the community page you will anyway need to pull all the configs (the addons themselves can be cached)